packages/@aws-cdk/cx-api/lib/versioning.ts

```diff
@@ -27,13 +27,13 @@ export function verifyManifestVersion(manifetVersion: string) {
 
   // if framework > cli, we require a newer cli version
   if (semver.gt(frameworkVersion, toolkitVersion)) {
-    throw new Error(`CDK CLI >= ${frameworkVersion} is required to interact with this app`);
+    throw new Error(`CDK CLI >= ${frameworkVersion} is required to interact with this app (got ${toolkitVersion})`);
   }
 
   // if framework < cli, we require a newer framework version
   if (semver.lt(frameworkVersion, toolkitVersion)) {
     throw new Error(
-      `CDK CLI can only be used with apps created by CDK >= ${CLOUD_ASSEMBLY_VERSION}`);
+      `CDK CLI can only be used with apps created by CDK >= ${toolkitVersion} (got ${frameworkVersion})`);
```

Comment:
`frameworkVersion` and `toolkitVersion` do not represent the CDK version but rather the manifest version. The addition of "got ${x}" will be misleading (it will report the manifest version of the toolkit/framework instead of the actual version). I would recommend not adding it.

Reply:
We purposely used CDK version numbers as manifest version numbers so that we can generate error messages like this. The alternatives would be:

```
The assembly is too new, get a newer CLI.
The assembly has version 5, but this CLI can only read assembly version 3. Get a newer CLI.
```

Both of which are horrible in terms of what a user should do about it, or what versions they should be looking for.

Reply:
My reservation is only with the `got ${frameworkVersion}`. It will be confusing because it will not print the framework version but rather the cloud assembly version that the framework presents. Either we should find some other phrasing or avoid this part of the message. The first part is useful.

Reply:
Oh I see what you mean. Agreed.